storage/src/factory.rs

```diff
@@ -117,6 +117,10 @@ impl BlobFactory {
     ) -> IOResult<Arc<dyn BlobCache>> {
         let backend_cfg = config.get_backend_config()?;
         let cache_cfg = config.get_cache_config()?;
+        let user_io_batch_size = match config.get_rafs_config() {
```

Comment:
Better to unwrap to default 0 here.

Reply:
It is unwrapped to `default_user_io_batch_size()` now, if can't get `rafs_config`.
So now we set this value exactly the same as the master branch (set to hard coded 1M, for any case including the case that can't get `rafs_config`)

Reply:
Maybe try `let user_io_batch_size = config.get_rafs_config().ok().unwrap_or_else(default_user_io_batch_size)`.

Reply:
Changed to map_or_else
```rust
let user_io_batch_size = config
    .get_rafs_config()
    .map_or_else(|_| default_user_io_batch_size(), |v| v.user_io_batch_size)
    as u32;
```